Annex60/ThermalZones/ReducedOrder/EquivalentAirTemperature/BaseClasses/PartialVDI6007.mo

```diff
@@ -65,6 +65,18 @@ initial equation
    equivalent air temperature calculation is close to 0.
    If there are no walls, windows and ground at all, this might be
    irrelevant.", level=AssertionLevel.warning);
+
+equation
+  delTEqLW=(TBlaSky-TDryBul)*alphaRad/(alphaRad+alphaWallOut);
+  delTEqSW=HSol*aExt/(alphaRad+alphaWallOut);
+  if withLongwave then
+    TEqWin=TDryBul.+delTEqLWWin*abs(sunblind.-1);
```

Comment:
If `0 <= sunblind <= 1` according to the comment, why don't you simply use
```
TEqWin=TDryBul.+delTEqLWWin*(1.-sunblind);
```
Also, please use
```
 Modelica.Blocks.Interfaces.RealInput sunblind[n](
   each min=0,
   each max=1,
   each final unit="1")
    "Opening factor of sunblinds for each direction (0 - open to 1 - closed)"
```